app/main/views.py

```diff
@@ -99,6 +165,43 @@ def get_service(service_id):
     return jsonify(services=jsonify_service(service))
 
 
+@main.route('/services-archive/<int:service_archive_id>', methods=['GET'])
+def get_archived_service(service_archive_id):
```

Comment:
I think I've missed a trick - maybe the table should be archive. But I've not thought it all through yet - how do we archive supplier changes? Do we need to in the same way? Is it auditing?

Like the naming though - I'll take you at this and update.